### ISSUE TITLE
Remove redundant sign-in chooser

### ISSUE THREAD

**Issue by mitchhentges** [taskcluster/taskcluster-web#399]
**Summary:**
Right now, there's only one way to Sign in: via credentials. Since, in my workflow, I've already made it explicit that I've wanted to sign in by clicking the button, and I've only got one sign in method, I think that I should go straight to the "Sign in with credentials" form

![2019-01-16-144911_460x223_scrot](https://user-images.githubusercontent.com/7784737/51283694-e55abe00-199d-11e9-965b-74582bcb51cf.png)

**Comment by varundey** [comment]
Wouldn't the fix for this discard #400 ? Also I would be happy to work on it

**Comment by helfi92** [comment]
> Wouldn't the fix for this discard #400 ?

No, it wouldn't. Taskcluster allows [multiple login strategies](https://github.com/taskcluster/taskcluster-web-server/blob/c5442223d2e6994a8bfc928afa13ae7a48ff4bc3/src/servers/createApp.js#L33-L38). For the time being we only have manual sign in however.

> Also I would be happy to work on it

Awesome, go for it :)

**Comment by helfi92** [comment]
How is it going @varundey? Let me know if you had questions for me.

**Comment by varundey** [comment]
I was confused and thought this is not an active issue since this repo was archived and thus this issue was read only some time back. I will probably start working on this on this coming weekend.

**Comment by helfi92** [comment]
Sounds good, thank you for the update. Just a heads up, you will want to fork and clone https://github.com/taskcluster/taskcluster.  This repo will be archived soon. The frontend code in the monorepo can be found under the `ui` directory.
